code/modules/client/preference_setup/antagonism/01_candidacy.dm

```diff
@@ -15,10 +15,10 @@
 	return list(":id" = pref.current_character)
 
 /datum/category_item/player_setup_item/antagonism/candidacy/gather_save_query()
-	return list("ss13_characters" = list("be_special_role", "id" = 1))
+	return list("ss13_characters" = list("be_special_role", "id" = 1, "ckey" = 1))
 
 /datum/category_item/player_setup_item/antagonism/candidacy/gather_save_parameters()
-	return list(":be_special_role" = list2params(pref.be_special_role), ":id" = pref.current_character)
+	return list(":be_special_role" = list2params(pref.be_special_role), ":id" = pref.current_character, ":ckey" = pref.client.ckey)
```

Comment:
This shit is necessary due to partial queries being a thing. Need to add character name in here the same way, if we want to make unique char names a thing.